### DISCUSSION TITLE
Styles not working on Netlify

### DISCUSSION THREAD

**Discussion by madman3** [thuliteio/doks#265] [Q&A] [answered]
Downloaded the source from https://github.com/h-enk/doks.git. Without changing anything, deployed it to netlify with a bitbucket repo. The styles are broken. I'm not sure what I'm doing wrong. I'm guessing its something very silly as I'm just getting my feet wet with the theme.

Its deployed here - https://608d44a98ee80a331e52c59c--amazing-sammet-858b4c.netlify.app/

I followed through the tutorial but no luck. Could you please help?

**Comment by h-enk** [comment, marked as answer]
Try changing `baseurl` in `./config/production/config.toml` 😉

**Reply by madman3** [reply]
Thank you so much, @h-enk! This worked for me :)

If I were to set up a custom domain, would I add the netlify url or the custom domain url as the baseurl?

**Reply by h-enk** [reply]
@madman3, the custom domain /the domain your website is available under (make it just one).

**Comment by anest-uk** [comment]
Someone who solved my very similar issue pointed me to this thread - I had the same thing https://github.com/h-enk/doks/discussions/280#discussion-3356871 .  I had customised _default/config.toml but not production/config.toml - even the tutorial I watched only mentions "your config.toml" and so I edited the wrong one/another one (there are three in total).  https://youtu.be/9Jqvaoeh1W4?t=1448 

BTW this thread is not marked answered... did h-enk solution not do it for you?

**Reply by madman3** [reply]
thanks for pointing me to that, @anest-uk. It's the same issue I had!

**Comment by drwilkins** [comment]
Thanks for the tips! Just to be absolutely clear about what fixed this for me:

My local development version worked perfectly with `npm start`, but when I pushed to github, triggering netlify deploy, it looked like this:
<img src="https://user-images.githubusercontent.com/6098768/136671279-e5317af0-e52a-4118-afb9-ea35fc1e035f.png" width= "400px">

Based on above suggestions, I added the following line to config> production> config.toml: 
`baseurl="https://my-random-netlify-url.netlify.app/"`

Commit, push, and voila:

<img src="https://user-images.githubusercontent.com/6098768/136671243-e6c1c19c-0ed5-4b2b-803a-d5372be27785.png" width="400px" >

**Comment by monicathieu** [comment]
This post also helped me!

I got turned a bit sideways because the [deployment page of the tutorial](https://getdoks.org/tutorial/deploy-your-site/) has the base URL variable spelled as `baseURL`, while `baseurl` (all lowercase) is what ended up working for me based on following the instructions in this thread. Would you be able to update that? 😅 

PS Thanks for all your work; the theme is lovely!

**Comment by T3TSV0** [comment]


**Reply by h-enk** [reply]
Do you have a repo to share?
